hexrd/ui/mask_manager_dialog.py

```diff
@@ -245,7 +251,7 @@ def export_visible_masks(self):
             for i, (det, mask) in enumerate(data):
                 parent = d.setdefault(det, {})
                 parent.setdefault(name, {})[str(i)] = mask
-        self.export_masks(d)
+        return d
```

Comment:
I think it also makes sense for `export_visible_masks()` to export the masks, rather than return the masks to be exported.